### ISSUE TITLE
Example project could use prebuilt pods/workspace 

### ISSUE THREAD

**Issue by RamblinWreck77** [jonkykong/SideMenu#122]
Hey, love your work!

I wanted to setup the example and noticed the pods/project/workspace weren't a part of the example folder so I went ahead and made them. 

It's not a huge deal, just simplifies the setup process to try out SideMenu and really see what it can do by being able to simply click the workspace, compile, and hop right in. 

## New Issue Checklist

I have read the [guidelines for contributing](https://github.com/jonkykong/SideMenu/blob/master/.github/CONTRIBUTING.md) and I understand:
- [ ] My issue was **not** solved in the [README](https://github.com/jonkykong/SideMenu/blob/master/README.md).
- [ ] My issue can **not** be answered on [stackoverflow.com](stackoverflow.com).
- [ ] My issue is **not** a request for new functionality that I am unwilling to build and contribute with a pull request.
- [ ] My issue **is** reproducible in the [demo project](https://github.com/jonkykong/SideMenu/tree/master/Example).

**Comment by jonkykong** [comment]
Hi @RamblinWreck77, thanks for proposing this. There are a lot of changes here, so I would like to understand the problem better to make sure this is the best solution. First, how did you get the library? Download, Cocoapods, or Carthage?

One of the largest problems myself and others had to deal with in this library is that updates don't seem to make it to Carthage and Cocoapods at the same time. Most recently, this was due to multiple copies of the source library being in the repo so that both Carthage and Cocoapods worked after install. This required multiple copies of source files to be updated which is undesirable and prone to error.

I want to ensure that when I update this repo, I can make a submission to Github and Cocoapods and everything else just *works*. To accomplish this I refactored the copies of source files mentioned earlier into links on a single copy.

Your changes look like they're relying on me to run Cocoapods against the project to fetch the source, as well as they're adding in copies of the source code again to the project...

**Comment by RamblinWreck77** [comment]
Hey,

Yeah that's exactly what I did: loaded the example folder, setup the podfile, ran pod install, then saved the workspace file. 

I then nuked it and copied my work back into a new folder to test the "just works" part and Xcode was immediately able to open everything, compile, and load your demo app on the device.

I see what you mean about having to redo this step for every update... may be more trouble then you want to go to. The goal of this was just to make it easier for someone considering your project to download > click > run, without needing to configure a podfile or anything else. 

I'm a newbie so if this is a bad idea my apologies for the issue/pull request. Maybe it'd just be easier to maintain an "example" fork and link it in your setup instructions?

-Eric
